Part-1-Functions

```diff
@@ -81,6 +99,12 @@ def end_other(a, b):
 # doubleChar('Hi-There') → 'HHii--TThheerree'
 
 def doubleChar(str):
+    c = ''              
+    for char in str:
+        c = c + char*2
+    return print(c)
+
+double(str)
   # CODE GOES HERE
```

Comment:
Dude stop using keyword 'str' its an inbuilt keyword, excluding this changes the function is correct